docs/usage-js.rst

```diff
@@ -26,48 +26,42 @@ As you can see, ``dredd.run`` is a function receiving another function as a call
 
 .. code-block:: javascript
 
    {
-     server: 'http://127.0.0.1:3000/api', // your URL to API endpoint the tests will run against
-     options: {
-       path: [],         // Required Array if Strings; filepaths to API description documents, can use glob wildcards
-       'dry-run': false, // Boolean, do not run any real HTTP transaction
-       names: false,     // Boolean, Print Transaction names and finish, similar to dry-run
-       loglevel: 'warning', // String, logging level (debug, warning, error, silent)
-       only: [],         // Array of Strings, run only transaction that match these names
-       header: [],       // Array of Strings, these strings are then added as headers (key:value) to every transaction
-       user: null,       // String, Basic Auth credentials in the form username:password
-       hookfiles: [],    // Array of Strings, filepaths to files containing hooks (can use glob wildcards)
-       reporter: ['dot', 'html'], // Array of possible reporters, see folder lib/reporters
-       output: [],       // Array of Strings, filepaths to files used for output of file-based reporters
-       'inline-errors': false, // Boolean, If failures/errors are display immediately in Dredd run
-       require: null,    // String, When using nodejs hooks, require the given module before executing hooks
-       color: true,
-     },
+     endpoint: 'http://127.0.0.1:3000/api', // your URL to API endpoint the tests will run against
+     path: [],         // Required Array if Strings; filepaths to API description documents, can use glob wildcards
+     'dry-run': false, // Boolean, do not run any real HTTP transaction
+     names: false,     // Boolean, Print Transaction names and finish, similar to dry-run
+     loglevel: 'warning', // String, logging level (debug, warning, error, silent)
+     only: [],         // Array of Strings, run only transaction that match these names
+     header: [],       // Array of Strings, these strings are then added as headers (key:value) to every transaction
+     user: null,       // String, Basic Auth credentials in the form username:password
+     hookfiles: [],    // Array of Strings, filepaths to files containing hooks (can use glob wildcards)
+     reporter: ['dot', 'html'], // Array of possible reporters, see folder lib/reporters
+     output: [],       // Array of Strings, filepaths to files used for output of file-based reporters
+     'inline-errors': false, // Boolean, If failures/errors are display immediately in Dredd run
+     require: null,    // String, When using nodejs hooks, require the given module before executing hooks
+     color: true,
      emitter: new EventEmitter(), // listen to test progress, your own instance of EventEmitter
      apiDescriptions: ['FORMAT: 1A\n# Sample API\n']
    }
 
+.. note::
+   The usage of nested `options` key is deprecated. Please list options under the root of the configuration. 
```

Comment:
I think this should be `warning`

Reply:
I'm missing a sentence about the `server`. Perhaps something like:

```restructuredtext
The former top-level ``server`` property should be now specified as ``endpoint``, not to be confused with the :option:`--server` option.
```

Also, beware, reST does

```restructuredtext
``options``
```

not

```markdown
`options`
```

Reply:
Adjusted.